common/djangoapps/xblock_django/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Looks like modernize didn't group these imports together in the migrations files. Do you mind grouping these together in all of the migrations files?

Reply:
My bad, changes are updated :)